backend/views/syson-diagram-general-view/src/main/java/org/eclipse/syson/diagram/general/view/services/GeneralViewNodeToolSectionSwitch.java

```diff
@@ -68,6 +72,12 @@ public Void casePartDefinition(PartDefinition object) {
         return super.casePartDefinition(object);
     }
 
+    @Override
```

Comment:
I would prefer `this.createToolsForCompartmentItems(object);` be added in Definition and Usage cases only.
Otherwise, when a caseXXX is added in a Switch class, the general rule is to add your case in alphabetical order. For the specific case of Element, it should be added in first position because it is the most generic case.

Reply:
done